cvxpylayers/torch/test_cvxpylayer.py

```diff
@@ -87,10 +87,9 @@ def test_least_squares(self):
 
         def lstsq(
             A,
-            b): return torch.solve(
-            (A_th.t() @ b_th).unsqueeze(1),
-            A_th.t() @ A_th +
-            torch.eye(n).double())[0]
+            b): return torch.linalg.solve(
+            A.t() @ A + torch.eye(n).double(),
```

Comment:
```suggestion
            A.t() @ A + torch.eye(n, dtype=torch.float64),
```

Reply:
Thanks, Parth! I'll make these changes locally and push them up with the updated example right now.